nailgun/entities.py

```diff
@@ -2740,17 +2870,41 @@ def read(self, entity=None, attrs=None, ignore=()):
         return super(PuppetModule, self).read(entity, attrs, ignore)
 
 
-class Realm(Entity):
+class Realm(
```

Comment:
Was you able to create 'Realm' entity with code we have here?
When I tried I get: ```<422: "URL is invalid - only  http://, https:// are allowed">```

If I try to create SmartProxy entity I get:
```
"Unable to communicate with the proxy: ERF12-2530 [ProxyAPI::ProxyException]: Unable to detect features ([RestClient::RequestTimeout]: Request Timeout) for proxy http://test.org/features","Please check the proxy is configured and running on the host."]
```

Maybe, I need full set of your changes :). Appreciate your feedback on that